src/components/_cards/CellarCard/AboutCellar.tsx

```diff
@@ -22,91 +24,107 @@ import { useDailyChange } from "data/hooks/useDailyChange"
 import { useTokenPrice } from "data/hooks/useTokenPrice"
 import { PercentageText } from "components/PercentageText"
 import { useIntervalGainPct } from "data/hooks/useIntervalGainPct"
+import { useCountdown } from "data/hooks/useCountdown"
+import { CellarKey } from "data/types"
 
 interface Props {
   data: CellarCardData
 }
 
 export const AboutCellar: React.FC<Props> = ({ data }) => {
   const cellarConfig = cellarDataMap[data.cellarId].config
+  const launchDate = cellarDataMap[data.cellarId].launchDate ?? null
   const { data: tvm } = useTvm(cellarConfig)
   const { data: apy, isLoading: apyLoading } = useApy(cellarConfig)
   const { data: activeAsset } = useActiveAsset(cellarConfig)
   const { data: cellarCap } = useCellarCap(cellarConfig)
   const { data: currentDeposits } = useCurrentDeposits(cellarConfig)
   const intervalGainPct = useIntervalGainPct(cellarConfig)
+  const isSteady = cellarConfig.cellar.key === CellarKey.PATACHE_LINK
+  const countdown = useCountdown({
+    launchDate,
+  })
 
   const tokenPrice = useTokenPrice(cellarConfig)
   const dailyChange = useDailyChange(cellarConfig)
   return (
     <>
-      <Stack mx={2} spacing={1}>
-        <Stack spacing={0}>
-          {isTVMEnabled(cellarConfig) && (
+      {!countdown && (
+        <Stack mx={2} spacing={1}>
+          <Stack spacing={0}>
+            {isTVMEnabled(cellarConfig) && (
+              <CellarStats
+                tooltip="Total value managed by Strategy"
+                title="TVM"
+                value={tvm?.formatted || "..."}
+                size="md"
+              />
+            )}
+
+            {isAPYEnabled(cellarConfig) && (
+              <CellarStats
+                tooltip={apy?.apyLabel || "..."}
+                title="APY"
+                value={apy?.expectedApy || "..."}
+                isLoading={apyLoading}
+              />
+            )}
+          </Stack>
+
+          {isTokenPriceEnabled(cellarConfig) && (
             <CellarStats
-              tooltip="Total value managed by Strategy"
-              title="TVM"
-              value={tvm?.formatted || "..."}
+              tooltip="The dollar value of the ETH, BTC, and USDC that 1 token can be redeemed for"
```

Comment:
this is still using clear gate cellar content, branch the ui content in `uiConfig.ts`